subsys/bluetooth/services/fast_pair/use_case/Kconfig

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menu "Fast Pair use case configuration"
+
+choice BT_FAST_PAIR_USE_CASE
+	prompt "Use case selection"
+	default BT_FAST_PAIR_USE_CASE_UNKNOWN
+	help
+	  Select the Fast Pair use case to configure the Fast Pair feature and extension set
+	  that satisfies the device feature requirements from the Google Fast Pair specification:
+	  https://developers.google.com/nearby/fast-pair/specifications/devicefeaturerequirement
+	  The use case selection should match the device type selection for your Fast Pair Model
+	  in the Google Nearby Console.
+
+config BT_FAST_PAIR_USE_CASE_UNKNOWN
+	bool "Unknown use case"
+	help
+	  Select the unknown Fast Pair use case. This use case configuration is neutral and is the
+	  default choice selection. The unknown use case does not enable any Fast Pair features and
+	  extensions. It also does not impose any restrictions on the Fast Pair Kconfigs.
+
+config BT_FAST_PAIR_USE_CASE_LOCATOR_TAG
+	bool "Locator Tag use case"
+	select BT_FAST_PAIR_FMDN
```

Comment:
If we want to enforce proper configuration, we may also need to forbid enabling features not supported for a given use-case (this would require using `depends on` syntax for the feature Kconfigs).

Alternatively, we can `imply`/`default y` features required for a given use case and let user to overwrite values for them (assuming that user is aware of the requirements and understand the consequences).

Reply:
https://nordicsemi.atlassian.net/browse/NCSDK-28098?focusedCommentId=1892197